### PR TITLE
migrated from true/false flag to counter

### PR DESCRIPTION
**Pull request by partamonov**
We had an issue if list was included multiple times and true/false condition was overwritten by last result.

### CODE REVIEW

File: cmd.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func cmdSearch(c *cli.Context) error {
 	for _, config := range configs {
 		log.Printf("Executing search for network list ID %s, Name %s in %s configuration", nlID, c.String("name"), config.ConfigName)
 
-		var rpFound, mtFound, spFound bool
+		var rpFound, mtFound, spFound int
```
Comment (RafPe) [inline cmd.go:35]: make some comments about what does counter mean

File: cmd.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ func cmdSearch(c *cli.Context) error {
...
 
 		//cMap used to append all information for Network list usage in one config
```
Comment (RafPe) [inline cmd.go]: Remove the comments :D

File: cmd.go (hunk 1 above)
Comment (partamonov) [reply cmd.go:35]: Done

File: cmd.go (hunk 2 above)
Comment (partamonov) [reply cmd.go]: Done

**Pull request merged**